frontend_v2/src/app/components/publiclists/challengelist/challengelist.component.ts

```diff
@@ -412,4 +420,25 @@ export class ChallengelistComponent implements OnInit {
       () => {}
     );
   }
+
+  fetchUnapprovedChallengesFromApi(id) {
+    const path = 'challenges/challenge_host_team/' + id + '/challenge';
```

Comment:
Can you please use this API url from `endpointsService` like we are doing at other places? @Kajol-Kumari